control-plane-operator/controllers/hostedcontrolplane/konnectivity/reconcile.go

```diff
@@ -100,6 +100,8 @@ func buildKonnectivityServerContainer(image string) func(c *corev1.Container) {
 			strconv.Itoa(KonnectivityServerPort),
 			"--health-port=8092",
 			"--admin-port=8093",
+			"--mode=http-connect",
+			"--proxy-strategies=destHost,default",
```

Comment:
I think you want this to be defaultRoute

Reply:
And default actually should never be specified as it's not desired in any situation (default is send to any random backend)

Reply:
defaultRoute deployment strategy:
https://github.com/kubernetes-sigs/apiserver-network-proxy/commit/554a60b5bacec1e3c27f218d10ec004a244b1719#diff-4ec4a95f03570d32223e630425a6daf105600a2bc4555dd38374307d09a67f30R47

(not default)